src/screens/FeedbackScreen.js

```diff
@@ -1,141 +1,183 @@
+                validate
+                rules={{ required: texts.feedbackScreen.inputsErrorMessages.name }}
+                errorMessage={errors.name && errors.name.message}
+                control={control}
+                containerStyle={styles.containerStyle}
+              />
+
+              <Input
+                name="email"
+                label={texts.feedbackScreen.inputsLabel.email + ' *'}
+                boldLabel
+                placeholder={texts.feedbackScreen.inputsLabel.email}
+                keyboardType="email-address"
+                validate
+                rules={{
+                  required: texts.feedbackScreen.inputsErrorMessages.email,
+                  pattern: {
+                    value: EMAIL_REGEX,
+                    message: texts.feedbackScreen.inputsErrorMessages.email
+                  }
+                }}
+                errorMessage={errors.email && errors.email.message}
+                control={control}
+                containerStyle={styles.containerStyle}
+              />
+
+              <Input
+                name="phone"
+                label={texts.feedbackScreen.inputsLabel.phone + ' *'}
+                boldLabel
+                placeholder={texts.feedbackScreen.inputsLabel.phone}
+                keyboardType="phone-pad"
+                validate
+                rules={{ required: texts.feedbackScreen.inputsErrorMessages.phone }}
+                errorMessage={errors.phone && errors.phone.message}
+                control={control}
+                containerStyle={styles.containerStyle}
+              />
+
+              <Input
+                name="message"
+                label={texts.feedbackScreen.inputsLabel.message + ' *'}
+                boldLabel
+                placeholder={texts.feedbackScreen.inputsLabel.message}
+                keyboardType="phone-pad"
+                multiline
+                textAlignVertical="top"
+                validate
+                rules={{ required: texts.feedbackScreen.inputsErrorMessages.message }}
+                errorMessage={errors.message && errors.message.message}
+                control={control}
+                containerStyle={styles.containerStyle}
+                inputStyle={styles.textArea}
+              />
+
+              <Controller
+                name="consent"
+                rules={{ required: texts.feedbackScreen.inputsErrorMessages.checkbox }}
+                render={({ onChange, value }) => (
+                  <>
+                    <Checkbox
+                      boldTitle
+                      title={texts.feedbackScreen.inputsLabel.checkbox + ' *'}
+                      checkedIcon="check-square-o"
+                      checkedColor={colors.accent}
+                      uncheckedIcon="square-o"
+                      uncheckedColor={colors.darkText}
+                      checked={value}
+                      onPress={() => onChange(!value)}
+                    />
```

Comment:
after submission, everything except the checkbox is resetted

<img width="374" alt="image" src="https://user-images.githubusercontent.com/1942953/232495828-b464c211-2ccc-4001-ab11-810d8a584c21.png">

please see that the checkbox value gets resetted as well

Reply:
The hidden input we created prevents the checkbox from resetting for some reason. So I suggest to remove the hidden input and raise an alert if the checkbox is unchecked. 

https://github.com/smart-village-solutions/smart-village-app-app/pull/741/commits/d28a8ed30bf4698775757852e5230ba93c8bdde4